pypowerwall/tedapi/__init__.py

```diff
@@ -103,12 +111,13 @@ def wrapper(*args, **kwargs):
 
 # TEDAPI Class
 class TEDAPI:
-    def __init__(self, gw_pwd: str, debug: bool = False, pwcacheexpire: int = 5, timeout: int = 5,
-                 pwconfigexpire: int = 5, host: str = GW_IP) -> None:
+    def __init__(self, gw_pwd: str, debug: bool = False, pwcacheexpire: int = 5, poolmaxsize: int = 10,
+                 timeout: int = 5, pwconfigexpire: int = 5, host: str = GW_IP) -> None:
```

Comment:
For API consistency (parameter location) for the TEDAPI initializer, can we move the new poolmaxsize parameter to the end?